swift/Arrow/Sources/Arrow/ArrowArray.swift

```diff
@@ -14,26 +14,25 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
```

Comment:
Is it intentional change?

Reply:
It is not, I will add the space back in.